run_jobs.py

```diff
@@ -110,11 +110,9 @@ def run_jobs(self) -> None:
 
     if how_started == 'DOCKER_COMPOSE':
         # Wait for MySQL container to finish setting up
-        logger.info('Waiting for the MySQL snail')
         # If it's not ready in two minutes, exit
-        num_loops = 40
+        num_loops = 41
         for i in range(1, num_loops + 1):
```

Comment:
nit: Rather than increasing the loop count to 41 (which I neglected to account for when I suggested moving the `sleep()` call), why not just let the range start with zero instead of 1…

```suggestion
        num_loops = 40
        for i in range(num_loops + 1):
```

Reply:
Yeah, this is better. Change made.